src/Utils/Strings.php

```diff
@@ -99,10 +99,13 @@ public static function substring(string $s, int $start, int $length = null): str
 
 
 	/**
-	 * Removes special controls characters and normalizes line endings and spaces in UTF-8 string.
+	 * Removes special controls characters and normalizes line endings and converts to NFC form and spaces in UTF-8 string.
 	 */
 	public static function normalize(string $s): string
 	{
+		// normalize string into utf8 NFC form
```

Comment:
* „UTF-8“ instead of „utf8“, or maybe even better to omit altogether
* the word „string“ is unnecessary

How about sth like „convert to NFC normal form “?

Reply:
👍

Reply:
fixed in b6dd526